### ISSUE TITLE
Responsiveness - Input

### ISSUE THREAD

**Issue by CrackerJax** [IroncladGames/SinsOfADarkAge#906]
Lately I've been having an issue with the client not picking up my clicks/ability casts etc. Not sure if it is ping related or something else. Bulwark especially seems to be delayed similar to how we had an issue back in the day. After reinstalling the client the problem still persists.

**Comment by CrackerJax** [comment]
Manny, i was typing when you labeled this wtf... lol

**Comment by uncannymanny** [comment]
Manny is legion.

**Comment by writor** [comment]
Random theory.

Do you think this could be tied to the "Quickcast Enabled on Up" settings that were recently introduced? If this is toggled on, abilities won't trigger until you release the key (introducing a short delay).

Granted, that wouldn't explain your hero just standing there and completely failing to respond to abilities. It sounds like maybe commands aren't being communicated to the server...

@jseward

**Comment by writor** [comment]
Linking to the old issue as well: #687

**Comment by CrackerJax** [comment]
@jseward I do not think it has to do with quickcast, as even when clicked on the abilities would not work or would delay. Also, I've gotten "stuck" - unable to move twice today - once in the forest, once while in lane. While I was able to chat - and move the camera - and see others still playing - my hero wouldnt respond to commands. It was odd.
